addons/project_purchase/models/project_project.py

```diff
@@ -36,6 +36,11 @@ def action_open_project_purchase_orders(self):
             'context': {
                 'default_project_id': self.id,
             },
+            'help': "<p class='o_view_nocontent_smiling_face'>%s</p><p>%s</p>" % (
+                _("No purchase order found. Let's create one."),
+                _("Once you ordered your products to your supplier, confirm your request for quotation and it will turn \
```

Comment:
I think the correct sentence is 'Once you ordered your products _from_ your supplier, [...]'

Reply:
Indeed, nice catch! :muscle: